.github/workflows/linting.yml

```diff
@@ -0,0 +1,101 @@
+name: Linting
+
+on:
+  pull_request:
+
+jobs:
+  PHPCompatibility:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        php: ['7.4']
```

Comment:
should we be prepping this for PHP8?

Reply:
Yeah, we should, but that's failing right now due to incompatible Composer dependencies. I added an issue about it so we can fix it later: #5 

I had 8.1 and 8.0 in the initial PR but backed them out due to failures for now.